File: go/iceman.go
package main

import (
	"fmt"
)

func main() {
	var n int
	fmt.Scan(&n)
	if n>100 {
		fmt.Println("Steam")
	} else if n<0 {
		fmt.Println("Ice")
	} else {
		fmt.Println("Water")
	}

}

File: nodejs/iceman.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});
readline.question("",n => {
    iceManAnswer(parseInt(n))
    readline.close();
});

function iceManAnswer(n) {
    if (n > 100)
        console.log("Steam")
    else if (n < 0)
        console.log("Ice")
    else
        console.log("Water")
    return
}